src/app/templates/_.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015-loose"],
```

Comment:
It also appears that [babel-preset-es2015-loose](https://www.npmjs.com/package/babel-preset-es2015-loose) is deprecated and now built into [babel-preset-es2015](https://www.npmjs.com/package/babel-preset-es2015) as an option - `{loose: true}`.